test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1369,15 +1369,12 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			Expect(err).ToNot(HaveOccurred())
 
 			By("verify the pod keep on pending")
-			err = wait.With(fxt.Client).Interval(10*time.Second).Steps(5).WhileInPodPhase(context.TODO(), pod.Namespace, pod.Name, corev1.PodPending)
+			_, err = wait.With(fxt.Client).Interval(10*time.Second).Steps(5).WhileInPodPhase(context.TODO(), pod.Namespace, pod.Name, corev1.PodPending)
```

Comment:
I think we should get `updatedPod` here:
`updatedPod, err = wait.With(fxt.Client).Interval(10*time.Second).Steps(5).WhileInPodPhase(context.TODO(), pod.Namespace, pod.Name, corev1.PodPending)`

Reply:
there is no need for the updatedPod in the rest of the test, we mainly care about what ns and name of that pod to do the checks and it is the same for pod and updated one. if that was a deployment it would be different but for orphan pods I didn't see the need

Reply:
uhm, for my own personal taste it would be clearer to use the `updatedPod`, but this is indeed mostly a personal preference so let's keep it like you did